templates/postgresql.conf-9.2.j2

```diff
@@ -39,11 +39,23 @@
 # The default values of these variables are driven from the -D command-line
 # option or PGDATA environment variable, represented here as ConfigDir.
 
-data_directory = '{{postgresql_data_directory}}'		# use data in another directory
+data_directory = {% if postgresql_docker_mode %}
```

Comment:
as discussed, I propose todo the binary choice (if docker_mode or not) once per variable used below (using set_fact in the tasks), so we can get rid of the ugly if/else here in the templates